### PR TITLE
Adds optional seed kwarg for reproducibility

### PR DESCRIPTION
**Pull request by alexgann**
@divamgupta 's Diffusion Bee UI suffered from not being able to leverage a seed to get reproducible results.

A large portion of prompt engineering/tweaking is being able to get the same base result and then iterate on the prompt. 

This PR simply adds an optional seed param - tested and works fine for getting reproducible results when given.

Thanks @fchollet and @divamgupta for the great work!

### CODE REVIEW

**Review by fchollet — APPROVED** [review]
LGTM, thanks!

**Comment by clavidk** [comment]
@alexgann @fchollet - I've been passing a seed param (using the colab), but not able to reproduce results even though using same seed.

5628FA43-1359-4A72-B982-53A51077C6FA

**Comment by alexgann** [comment]
@clavid87 - what codebase are you using in your notebook there?  I don't think this fork provides an interface method named `text_to_image`?

**Comment by clavidk** [comment]
@alexgann Hmm it's this colab made by @fchollet: https://colab.research.google.com/drive/12l_Wq4km2nLXHZDo3RyzeFTnTnXq0aiu?usp=sharing#scrollTo=ajKqSj970dOp (I thought it was the same one referenced in the readme of this fork but it appears that it's not, so not sure how I came to this colab)

Regardless, you can see it's using `keras_cv.models.StableDiffusion`

**Comment by alexgann** [comment]
@clavid87 - got it.  Loaded up the colab - looks like you need to prepend `keras.utils.set_random_seed(101)` in the cell you're executing to generate the image:
image

After doing that I was able to get the same result each execution.  Let me know how that works for you.
